src/openloco/windows/options.cpp

```diff
@@ -50,7 +50,7 @@ namespace openloco::ui::options
     {
         auto widget = &w->widgets[index];
 
-        gfx::point_t pos = {};
+        gfx::point_t pos = gfx::point_t();
```

Comment:
Any particular reason for this change?

Reply:
Good question. Let me revert and see why I did that.